common/api/imodel-components-react.api.md

```diff
@@ -333,7 +333,7 @@ export interface CubeProps extends React_2.AllHTMLAttributes<HTMLDivElement>, Co
 }
 
 // @public
-export class CubeRotationChangeEvent extends UiEvent<CubeRotationChangeEventArgs> {
+export class CubeRotationChangeEvent extends BeUiEvent<CubeRotationChangeEventArgs> {
```

Comment:
The same arguments apply to "removal of UiEvent" since `category` property is not on `BeUiEvent` and a different subclass is used which would be breaking in the case of `instanceof` checks.

Reply:
Category is added to `UIError`, not `UiEvent`, maybe that's the confusion :)

Reply:
Yes, nonetheless, the subclass instanceof check would break users. I did look into https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/hasInstance which could be used to keep those checks in tact, but I think it's easier to wait for the next major release.